Week1/connection-test.js

```diff
@@ -1,9 +1,11 @@
 var mysql      = require('mysql');
 var connection = mysql.createConnection({
   host     : 'localhost',
-  user     : 'hyfuser',
+  user     : 'Fatma',
   password : 'hyfpassword',
-  database : 'userdb'
+  database : 'userdb',
+  // port : 3307 // Uncomment this line and replace xxxx with the selected port number if you are not using default 3306. I also suggest to download MySQL version 5.7 because recent versions has authentication problems
+
 });
```

Comment:
this file should not be a part of the PR / commit

Reply:
Done! I delete it from next commit